c++/src/capnp/any.h

```diff
@@ -221,9 +221,14 @@ struct AnyPointer {
     inline void setAs(std::initializer_list<ReaderFor<ListElementType<T>>> list);
     // Valid for T = List<?>.
 
+    template <typename T>
+    inline void setAsCanonical(ReaderFor<T> value);
```

Comment:
Should be `setCanonicalAs`, since the `As` is meant to refer to the template parameter type.